doc/setup_instructions.md

```diff
@@ -67,41 +73,44 @@ You need a supported development environment for Godot with
 mono. Note: it is possible to get by with just C# build tools, but
 installing a development environment is the easier route.
 
+Godot currently supports the following development environments:
+
+- Visual Studio 2019
+- Visual Studio Code
+- MonoDevelop
+- Visual Studio for Mac
+- JetBrains Rider
+
+### MonoDevelop
+
 On Linux MonoDevelop is recommended. To get an up to date version,
 first enable the mono repository:
 https://www.mono-project.com/download/stable/ and then install the
 following packages with your package manager: `mono-complete
-monodevelop nuget`
+monodevelop nuget`. Make sure it is a newever version of mono that
```

Comment:
There's still a typo on this line? `newever`
Could you maybe also not force push so it is easier to see what gets fixed in each commit? I'm going to squash this PR anyway so it doesn't matter.